Scripts/script-DeleteConext.yml

```diff
@@ -30,7 +31,11 @@ script: |
           return "Context cleared";
       }
   } else {
-      return executeCommand('delContext', { key: args.key });
+      var key = args.key;
+      if (isSubPlaybookKey) {
+        key = 'subplaybook-${currentPlaybookID}.' + key;
```

Comment:
Are you sure that it will be replaced correctly this way?
Basically we need something like:
${currentPlaybookID='subplaybook-'+val+'.' + key}

Reply:
Yes, replaced correctly